python/cudf/cudf/__init__.py

```diff
@@ -167,4 +173,7 @@
     "to_datetime",
     "to_numeric",
     "unstack",
+    "get_option",
+    "set_option",
+    "describe_option",
```

Comment:
This list is alphabetized except for these new entries. We could sort them, too.